pythonitalia/pycon
# Users widget for django admin

Linear issue with all the places where it should be used: https://linear.app/python-italia/issue/WEB-147/django-admin-microservice-utenti

# Why

Django admin needs to display more information about our users, just showing the id is not useful

<img width="538" alt="image" src="https://user-images.githubusercontent.com/3382153/142131012-778a5491-81e6-4a44-a362-692ddf0ba795.png">

<img width="573" alt="image" src="https://user-images.githubusercontent.com/3382153/142131020-29519d5f-2cf7-459b-b406-b24cb3553135.png">

<img width="1113" alt="image" src="https://user-images.githubusercontent.com/3382153/142131051-3ea0959f-aa03-4cbd-883b-53e812cd3f26.png">


# How to test

Start docker setup
Go to localhost:3000/
Create some users

Go to the admin blog post http://localhost:8000/admin/blog/post/
open an existing blog post / create a new one

"Author" will now be an autocomplete field

Search the users you created it, it should match

Go over submission, schedule item and others they should all work
